.github/buildomat/jobs/opte-api.sh

```diff
@@ -24,7 +24,7 @@ header "check API_VERSION"
 ./check-api-version.sh
 
 header "check style"
-ptime -m cargo fmt -- --check
+ptime -m cargo +nightly fmt -- --check
```

Comment:
I feel like I'm asking something obvious, but: if we got most of the crates off nightly, why is it required here? Same goes for other places where `check` is called without the kernel feature.

Reply:
Just need it for `fmt` specifically because the `rustfmt.toml` we have uses some unstable options that need a nightly. The `check` ones that follow don't use nightly otherwise.